### DISCUSSION TITLE
Clone from URL (Repo is empty), but the property `bare` return false.

### DISCUSSION THREAD

**Discussion by dota17** [gitpython-developers/GitPython#1169] [Q&A] [answered]
Example, I just clone from URL : [https://github.com/apache/incubator-argus](https://github.com/apache/incubator-argus).
It is an empty repo. But when I call git.Repo.clone_from, the property `bare` return False.

Maybe it is a BUG.

**Comment by Byron** [comment, marked as answer]
I believe the issue here may be a misunderstanding of what [a bare repository](https://git-scm.com/docs/gitglossary.html#Documentation/gitglossary.txt-aiddefbarerepositoryabarerepository) is, as any repository can be a bare clone if it was cloned with `bare=true` or `--bare` respectively.
The `bare` flag has nothing to do with the repository being empty.

**Comment by dota17** [comment]
How we can use gitpython to check an empty repo?

**Comment by Byron** [comment]
You could indeed clone a bare repository and then have a look at the [tree at the branch of interest](https://gitpython.readthedocs.io/en/stable/tutorial.html#the-tree-object) - it represents the top-level directory.
